app.js

```diff
@@ -57,9 +58,8 @@ try {
 } catch (e) {
 	if (require.main !== module) throw new Error("Dependencies unmet");
 
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	child_process.spawnSync('npm', ['install', '--production'], {stdio: 'inherit'});
 }
```

Comment:
Does anyone object to removing this code entirely? `pokemon-showdown` already has it; anyone manually running `app.js` shouldn't need this.